tests/gcm/test_auto.py

```diff
@@ -19,7 +19,7 @@ def _generate_linear_regression_data():
 
 def _generate_non_linear_regression_data():
     X = np.random.normal(0, 1, (1000, 5))
-    Y = np.sum(X**2, axis=1)
+    Y = np.sum(np.log(abs(X)), axis=1)
```

Comment:
Is there a reason why moved to this generative model?

Reply:
Wanted to have non-linear data that cannot be capture by a model with polynomial features (here, `X**2` would be captured by it with a degree 2).